src/index.html

```diff
@@ -5,9 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Kosugi&family=Roboto:wght@300;400;500;700;900&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+      >
+
+      <img
+        src="images/imac.jpeg"
+        alt="imac image"
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))